components/timestamp/timestamp.tsx

```diff
@@ -90,6 +92,47 @@ function normalizeRangeDescriptor(unit: NonNullable<Props['units']>[number]): Ra
     return unit;
 }
 
+function timezoneDiff(timezone: string | undefined, currentTimezone: string | undefined): ReturnType< typeof FormattedMessage > | undefined {
+    if (!timezone) {
+        return undefined;
+    }
+
+    const currentDate = DateTime.local().setZone(currentTimezone);
+    const userDate = DateTime.local().setZone(timezone);
+
+    const hoursDiff = Duration.fromObject({
+        minutes: userDate.offset - currentDate.offset,
```

Comment:
@calebroseland - I went with calculating the offset in minutes and converting to hours because when using `fromDateTimes` it would assume if your timezone was `UTC` that there was zero diff between current and user time. 

I can play with it some more of we want to only use Luxon's `.toHuman`

Reply:
To be specific, the alternative was

```ts
    const userDate = new Date(value.toLocaleString('en-US', {timeZone: timezone}));
    const currentDate = new Date(value.toLocaleString('en-US', {timeZone: currentTimezone}));

    const hoursDiff = Interval.fromDateTimes(userDate, currentDate).toDuration('hours');

    if (!hoursDiff.hours) {
        return undefined;
    }

    const aheadOrBehind = hoursDiff.hours < 0 ? 'ahead' : 'behind';

    if (aheadOrBehind === 'ahead') {
        return (
            <FormattedMessage
                id='user_profile.account.hoursAhead'
                defaultMessage=' ({hourDiff} ahead)'
                values={{
                    hourDiff: hoursDiff.toHuman({unitDisplay: 'narrow'}),
                }}
            />
        );
    }

```